docker-compose.yaml

```diff
@@ -15,6 +15,13 @@ services:
     network_mode: service:mainnetwork
     environment:
       - DJANGO_SECRET_KEY=secretkey
+  reports:
+    environment:
+      - DJANGO_SECRET_KEY=secretkey
+      - SKIP_FRONTEND_BUILD=yes
```

Comment:
I would opt to use one env variable "IS_REPORT_CONTAINER" for all, instead of 3 different skips, but either is ok.